scss/bitstyles/generic/_reset.scss

```diff
@@ -22,6 +22,10 @@ select {
   margin: 0;
 }
 
+ul {
+  padding: 0;
+}
```

Comment:
does `list-style-type` not matter anymore? and do we need this reset for ol's as well?

Reply:
list-style-type is still specified using a classname. This PR doesn’t really do much anything with that, though it does set the basis for the follow-up PR at #704 which changes the classname from being an atom-level component class to being a utility class

But yes, this should be applied to `ol`s too, thanks! I’ll make that change here, and update the other PR so there’s no duplication